lib/simctl.ts

```diff
@@ -37,7 +37,16 @@ export class Simctl implements ISimctl {
 	}
 
 	public getAppContainer(deviceId: string, applicationIdentifier: string): IFuture<string> {
-		return this.simctlExec("get_app_container", [deviceId, applicationIdentifier]);
+		return (() => {
+			try {
+				return this.simctlExec("get_app_container", [deviceId, applicationIdentifier]).wait();
+			} catch(e) {
+				if (e.message.indexOf("No such file or directory") > -1) {
```

Comment:
Another approach would be to enumerate the directory contents of `~/Library/Developer/CoreSimulator/Devices/[DeviceID]/data/Containers/Data/Application/`. AFAIK we've depricated this approach for Xcode 7.0+ and in my opinion it is much cleaner and easier to use `simctl`.